solr-plugin/pom.xml

```diff
@@ -0,0 +1,135 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>codice-maven</artifactId>
+        <groupId>org.codice.maven</groupId>
+        <version>0.3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>solr-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <name>solr-plugin Maven Mojo</name>
+    <url>http://maven.apache.org</url>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-plugin-api</artifactId>
+            <version>3.6.0</version>
```

Comment:
❗️ Versions should be defined in the top pom using properties. Also add these dependencies in its dependency management section along with the versions referencing those properties. Here, only reference the artifact without the version.